### ISSUE TITLE
New feature request: Convert a pdf in pdf/A

### ISSUE THREAD

**Issue by X1z1** [qpdf/qpdf#85]
Add in qpdf a simple command to convert a normal pdf in a pdf/A

**Comment by jberkenbilt** [comment]
I'm not sure what pdf/A is specifically. Is that with accessibility? This is very likely something that would be out of scope for qpdf itself. If that's not what you're asking for, please clarify. Thanks!

**Comment by X1z1** [comment]
For pdf/A i mean https://en.wikipedia.org/wiki/PDF/A
I think that PDF/A-1 format is enough

Some organizations ask pdf/A format for preservation of electronic documents.
But after i use qpdf to manipulate pdfs, they are no more in pdf/A format.
So i request a feature that can simply convert a pdf in a pdf/A, in this way that organizations accept the pdf/A (but not a normal pdf).

**Comment by jberkenbilt** [comment]
I'm afraid this is not possible with qpdf. Generating PDF/A will be the responsibility of the original PDF creation software. Converting a PDF file to PDF/A goes well beyond just content transformations. It would probably be possible for someone to use the qpdf library to check whether a PDF file is compliant, and qpdf could be used to perform some of the operations, but in general, this would be out of scope for what the qpdf library is designed to do.

**Comment by kobayashison** [comment]
It's an old issue but can be useful for others with the same need. [ocrmypdf](https://github.com/ocrmypdf/OCRmyPDF) can convert to PDF/A:
```
ocrmypdf --tesseract-timeout=0 --remove-background input.pdf output.pdf
```
https://ocrmypdf.readthedocs.io/en/latest/cookbook.html#don-t-actually-ocr-my-pdf

**Comment by jeromerobert** [comment]
For clarification... *ocrmypdf* mostly relies on qpdf (thanks to pikepdf) but PDF/A export is performed by Ghostscript (not by qpdf):
https://github.com/ocrmypdf/OCRmyPDF/blob/d9eb0ba7efff5ae691d84a6bec40b39b7ad593a6/src/ocrmypdf/_exec/ghostscript.py#L199

**Comment by jberkenbilt** [comment]
Since my original comment, changes were made to qpdf to _preserve_ PDF/A if the file is already PDF/A compliant, but qpdf still can't create PDF/A from a non PDF/A file.

**Comment by rubyFeedback** [comment]
Interesting. I just found this here when I searched for it, that is, qpdf + PDF/A; StackOverflow has only
examples with gs (ghostscript). My rationale for needing or wanting this may be newbie-ish in that
I am currently archiving pdf files from the 1990s to 2010 (the from 1990s were originally
stored on a floppy disc and later stored as some Microsoft office format; I then converted
this into .pdf but now I am trying to optimise it, keeping only relevant stuff in it after
optimising it for small size; and so I thought PDF/A was further useful in stripping away
things not needed as much. Perhaps this could be reviewed in the future, if qpdf could
generate PDF/A files too.)
